src/components/UserList.tsx

```diff
@@ -1,75 +1,53 @@
 import React from 'react';
 import clsx from 'clsx';
-import { createStyles, WithStyles, withStyles } from '@mui/styles';
-import { Accordion, AccordionDetails, AccordionSummary, CustomTheme } from '@mui/material';
+import { Box, Accordion, AccordionDetails, AccordionSummary, Typography } from '@mui/material';
 import { ExpandMore } from '@mui/icons-material';
-import Typography from '@mui/material/Typography';
 import ManageUsersView from './ManageUsersView';
 
-const styles = (theme: CustomTheme) =>
-  createStyles({
-    header: {
-      fontSize: '14px',
-      lineHeight: '22px',
-      fontWeight: '600',
-      color: theme.palette.primary.main,
-    },
-    expandIcon: {
-      color: theme.palette.primary.main,
-    },
-    canManageUsers: {
-      paddingTop: 0,
-    },
-    values: {
-      paddingBottom: theme.spacing(1),
-    },
-    root: {
-      marginTop: theme.spacing(3),
-    },
-    noUsers: {
-      fontStyle: 'italic',
-      colorPrimary: theme.palette.error.contrastText,
-      color: theme.palette.error.contrastText,
-      paddingBottom: theme.spacing(1),
-    },
-  });
-
-interface UserListProps extends WithStyles<typeof styles> {
+interface UserListProps {
   canManageUsers: boolean;
   defaultOpen?: boolean;
   removeUser?: (removableEmail: string) => void;
   typeOfUsers: string;
   users: Array<string>;
 }
 
-function UserList({
-  classes,
-  canManageUsers,
-  defaultOpen,
-  removeUser,
-  typeOfUsers,
-  users,
-}: UserListProps) {
+function UserList({ canManageUsers, defaultOpen, removeUser, typeOfUsers, users }: UserListProps) {
   return (
     <Accordion defaultExpanded={defaultOpen}>
       <AccordionSummary
-        expandIcon={<ExpandMore className={classes.expandIcon} />}
-        className={classes.header}
+        expandIcon={<ExpandMore sx={{ color: 'primary.main' }} />}
+        sx={{
+          fontSize: '14px',
+          lineHeight: '22px',
+          fontWeight: '600',
+          color: 'primary.main',
```

Comment:
Do these need to `import theme from 'modules/theme';` and use `theme.palette.primary.main` instead like in `src/components/DatasetView.tsx`? If so, a number of components in this PR have this issue with `color`.

Reply:
Great question - I believe it is correct as is - Here's[ the documentation](https://mui.com/system/getting-started/the-sx-prop/?srsltid=AfmBOor9PjYY4oizLrQBEB2KSea8tGUdEU5VACrtpu6Js6RSEbVKr837#basic-example) and the basic example has something very similar. We have ["ThemeProvider" set up in the index.](https://github.com/DataBiosphere/jade-data-repo-ui/blob/829daf5eb149d9896b9903ce9cf4c297e8090104/src/index.jsx#L162), so the theme should be available to all components in the repo. 

I'm possibly not referencing the theme correctly in DatasetView, but I wasn't quite sure how to do the expansion of the custom mixin.

Reply:
Does that also mean that it isn't necessary for `DatasetView.tsx`?